### PR TITLE
RA-1195 fix ManagePersonTest

### PR DESCRIPTION
**Pull request by AdamGrzybkowski**
Merge this https://github.com/openmrs/openmrs-contrib-uitestframework/pull/14 first

### CODE REVIEW

File: ui-tests/src/main/java/org/openmrs/reference/page/AddPersonPage.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package org.openmrs.reference.page;
```
Comment (rkorytkowski) [inline ui-tests/src/main/java/org/openmrs/reference/page/AddPersonPage.java]: Missing license.

File: ui-tests/src/main/java/org/openmrs/reference/page/AdministrationPage.java [hunk 2]
```diff
@@ -4,15 +4,13 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 
-/**
- * Created by tomasz on 10.07.15.
- */
 public class AdministrationPage extends Page {
```
Comment (rkorytkowski) [inline ui-tests/src/main/java/org/openmrs/reference/page/AdministrationPage.java]: No license.

Comment (AdamGrzybkowski) [inline ui-tests/src/main/java/org/openmrs/reference/page/AdministrationPage.java]: PR updated

**Comment by rkorytkowski** [comment]
Please resolve conflicts.

**Comment by AdamGrzybkowski** [comment]
@rkorytkowski  done

**Comment by rkorytkowski** [comment]
All good. Please resolve conflicts and merge.

**Pull request merged**